app/controllers/admin/users_controller.rb

```diff
@@ -2,6 +2,8 @@
 
 module Admin
   class UsersController < AdminController
+    before_action :validate_user, only: %i(make_admin, remove_admin, make_moderator, remove_moderator, make_spotlight, remove_spotlight)
+    before_action :validate_role, only: %i(make_admin, remove_admin, make_moderator, remove_moderator)
```

Comment:
- Rails/LexicallyScopedActionFilter: `make_admin,`, `remove_admin,`, `make_moderator,` are not explicitly defined on the class.
- Lint/PercentSymbolArray: Within `%i`/`%I`, ':' and ',' are unnecessary and may be unwanted in the resulting symbols.
- Style/PercentLiteralDelimiters: `%i`-literals should be delimited by `[` and `]`.